packages/api/tests/lambdas/test-publish-reports.js

```diff
@@ -394,6 +394,16 @@ test.serial('lambda publishes correct granules from payload.granules to SNS topi
     }
   });
 
+  await executionModel.create(
```

Comment:
A [recent change](https://github.com/nasa/cumulus/commit/5220a907a0ba647327bf1fc49b2e94de6ee0f368) was causing a 7-second timeout in a number of tests in this file. (If it tried to update an execution that did not already exist in the DB, it would wait 7 seconds for that item to appear in the DB.) By adding those records to the DB before running `publishReports.handler`, that timeout is avoided.

Reply:
Makes sense, though it's not necessary in this specific test, because it's just creating an execution in a "running" state so it doesn't attempt to look up a previous execution (that is only done for "completed"/"failed" executions)

Reply:
Right you are, I'll fix that.

Reply:
Fixed in 269cbae25